### DISCUSSION TITLE
How do you use format and displayFormat settings?

### DISCUSSION THREAD

**Discussion by craigcordell** [malzariey/filament-daterangepicker-filter#59] [Q&A] [answered]
Have tried a few times to change all displayed dates to m/d/Y. Have tried every combination that I can think of in php and JS date formats to achieve a display that shows month/day/year. All either display incorrectly or cause an error on the back end. Can anyone tell me what the ->format() and ->displayFormat() parameters should be to acheive this?

**Comment by craigcordell** [comment, marked as answer]
Please disregard. Must have had a typo. To show and use common United States date format of month/day/year ...

->displayFormat("MM/DD/YYYY") 
->format('m/d/Y')

**Reply by citricguy** [reply]
As a note for future searchers, if you don't want the leading 0's use

->displayFormat("M/D/YYYY")
->format('m/d/Y')
